query-node/joystream-query-node/schema.graphql

```diff
@@ -0,0 +1,16 @@
+type Membership {
+  # Member's root account id
+  accountId: String!
```

Comment:
How is the primary key in database defined from this schema?

Reply:
It's a good question. Here is the flow:

Schema parser creates _model definitions_ from the schema and then warthog cli use these model definitions to create models.
Warthog cli only understands `columnName:columnType` pattern, this means we can't decide to make a column unique or primary key at this point. And by default warthog adds `id` to each model.  

Here is an example of the warthog cli model generation:

`warthog generate Membership accountId! memberId:int!`